app/views/proctored_exams/show.erb

```diff
@@ -1,3 +1,17 @@
+<!-- # Copyright (C) 2017 Atomic Jolt
```

Comment:
Maybe move the copyright to the next line?
Remove the `#` in the erb files.
```erb
<!--
Copyright (C) 2017 Atomic Jolt

This program is free software: you can redistribute it and/or modify
it under the terms of the GNU Affero General Public License as published by
the Free Software Foundation, either version 3 of the License, or
(at your option) any later version.

This program is distributed in the hope that it will be useful,
but WITHOUT ANY WARRANTY; without even the implied warranty of
MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
GNU Affero General Public License for more details.

You should have received a copy of the GNU Affero General Public License
along with this program.  If not, see <http://www.gnu.org/licenses/>.
-->
```